docs/includes/_access_token_create_public_client_pkce.mdx

```diff
@@ -1,105 +1,105 @@
   ```
 
-  where:  
+  where:
 
-   - {client_id} is your app's **Client ID**  
+    - {client_id} is your app's **Client ID**
 
-   - {redirect_uri} is one of the app's configured **Redirect URI** values  
+    - {redirect_uri} is one of the app's configured **Redirect URI** values
 
-   - {scopes} is 'openid' plus one or more of the app's **Allowed Scopes**, space delimited  
+    - {scopes} is 'openid' plus one or more of the app's **Allowed Scopes**, space delimited
 
-   - {state} is a value generated by your app to maintain state betewen the request and response  
+    - {state} is a value generated by your app to maintain state betewen the request and response
 
-   - {codeChallenge} is generated as defined in [RFC 7636](https://datatracker.ietf.org/doc/html/rfc7636#section-4.2), example JavaScript snippet below:  
+    - {codeChallenge} is generated as defined in [RFC 7636](https://datatracker.ietf.org/doc/html/rfc7636#section-4.2), example JavaScript snippet below:
 
-   ```javascript
-     codeVerifier = crypto.randomBytes(32).toString('base64url');
-     codeChallenge = crypto.createHash('sha256').update(codeVerifier).digest().toString('base64url');
-   ```
+    ```javascript
+    codeVerifier = crypto.randomBytes(32).toString('base64url');
+    codeChallenge = crypto.createHash('sha256').update(codeVerifier).digest().toString('base64url');
+    ```
 
-3. Consume the '/authorize' response and authorization code  
+3. Consume the '/authorize' response and authorization code
 
-  In response to the '/authorize' request, Beyond Identity redirects to the app's configured authentication url. 
-  In this example, the authenticator configuration type is "Hosted Web", so the authentication url is hosted by Beyond Identity (see [Authenticator Configurations](/docs/next/authentication)).  
+  In response to the '/authorize' request, Beyond Identity redirects to the app's configured authentication url.
+  In this example, the authenticator configuration type is "Hosted Web", so the authentication url is hosted by Beyond Identity (see [Authenticator Configurations](/docs/next/authentication)).
 
   :::info important  
   Please note that the redirection must be handled in the front end (such as React, Angular, or vanilla JavaScript running in the user's browser) in order for passkey authentication to work.  
-  :::  
+  :::
 
-  Once passkey based authentication is complete, Beyond Identity will redirect back to the app's configued redirect URI with a 'code' parameter in the query string.  You will use this code in the '/token' request.  
+  Once passkey based authentication is complete, Beyond Identity will redirect back to the app's configued redirect URI with a 'code' parameter in the query string. You will use this code in the '/token' request.
 
+1. Create your '/token' request
```

Comment:
There's several places where I'd recommend using back ticks for consistency. I'll stop pointing them out to save space on the GitHub page :)

```suggestion
1. Create your `/token` request
```